NadekoBot.Core/Services/NadekoBot.cs

```diff
@@ -94,7 +94,8 @@ public NadekoBot(int shardId, int parentProcessId)
                 ConnectionTimeout = int.MaxValue,
                 TotalShards = Credentials.TotalShards,
                 ShardId = shardId,
-                AlwaysDownloadUsers = false
+                AlwaysDownloadUsers = false,
+                GatewayIntents = GatewayIntents.All
```

Comment:
You should add just the one intent or else the bot will get flooded with messages it doesn't need to handle

Reply:
The bot always had every single unprivileged intent by default. This just changes it to include 2 extra ones

Reply:
https://discordnet.dev/api/Discord.WebSocket.DiscordSocketConfig.html read here the default is allunprivileged

Reply:
The bot doesn't get flooded with messages it doesn't need to handle, it uses most of the intents

Reply:
ok fine, it won't be much worse but there are a ton of features we don't use. But trying to whittle down the intents would take too long